lib/auth.js

```diff
@@ -11,9 +10,24 @@ const ghauth = util.promisify(ghauthBase);
 
 export default lazy(auth);
 
+function errorExit(message) {
+  process.stderr.write(`${message}\n`);
+  process.exit(1);
+}
+
 function check(username, token) {
-  assert(typeof username === 'string' && /^[a-zA-Z0-9]*/.test(username));
-  assert(typeof token === 'string' && /^[0-9a-f]*/.test(token));
+  if (typeof username !== 'string') {
+    errorExit(`username must be a string, received ${typeof username}`);
+  }
+  if (!/^[a-zA-Z0-9]+$/.test(username)) {
```

Comment:
These are GitHub usernames, aren't they? They're allowed to contain hyphens, at least according to https://github.com/join:
![image](https://user-images.githubusercontent.com/5445507/159138227-fb5bbdcd-1415-4120-a090-195fc37da3b9.png)

Reply:
I added hyphens as a permissible character.